script/encrypt

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -ueo pipefail
+
+ROOT=$(cd $(dirname $0)/.. && pwd)
+
+source ${ROOT}/script/include/credentials.sh
+
+INSTANCE=$(credential instance)
+SRC=${ROOT}/credentials.yml
+DEST=${1:-${ROOT}}
+
+[ -d "${DEST}" ] && {
+  DEST="${DEST}/credentials-${INSTANCE}.yml.enc"
+}
+
+exec openssl enc -md sha1 -salt -aes-256-cbc -in ${SRC} -out ${DEST}
```

Comment:
_dons tinfoil hat_

* I would prefer to use `-md sha256` since SHA1 is considered kinda wimpy nowadays.
* `-salt` is used by default, so isn't necessary
* Having to provide the passphrase interactively kinda sucks and encourages weak passphrases. We could use `-pass env:PASSPHRASE` or `-pass stdin` to make this a little more palatable.
* I'm wondering about deriving the key/IV from a passphrase, and whether we should store the key in a variable and generate our own IV. Upon closer inspection, it looks like openssl is salting+hashing the passphrase in order to create a key of the correct length, which I guess is fine.